### DISCUSSION TITLE
Why replication webhook show so strange output?

### DISCUSSION THREAD

**Discussion by microyahoo** [goharbor/harbor#19957] [Q&A] [answered]
I follow the [configure webhooks guide](https://goharbor.io/docs/2.0.0/working-with-projects/project-configuration/configure-webhooks/) to configure replication wehbhooks, but receive so strange message of `name_tag: busybox [1 item(s) in total]`. Normally, tags information should be displayed. The [code](https://github.com/goharbor/harbor/blob/main/src/controller/event/handler/webhook/artifact/replication.go#L125-L126) looks okay to get name and tag.

```json
{
    "type":"REPLICATION",
    "occur_at":1707127076,
    "operator":"EVENT",
    "event_data":{
        "replication":{
            "harbor_hostname":"reg.deeproute.ai",
            "job_status":"Success",
            "description":"",
            "artifact_type":"artifact",
            "authentication_type":"basic",
            "override_mode":true,
            "trigger_type":"EVENT",
            "policy_creator":"zhihuanzhu",
            "execution_timestamp":1707127075,
            "src_resource":{
                "registry_type":"harbor",
                "endpoint":"https://reg.deeproute.ai",
                "namespace":"deeproute-public"
            },
            "dest_resource":{
                "registry_name":"Tencent",
                "registry_type":"docker-registry",
                "endpoint":"https://ccr.ccs.tencentyun.com",
                "namespace":"deeproute-tke"
            },
            "successful_artifact":[
                {
                    "type":"artifact",
                    "status":"Success",
                    "name_tag":"busybox [1 item(s) in total]"
                }
            ]
        }
    }
}
```

**Comment by stonezdj** [comment]
Could you please confirm the tag exists for the busybox artifact? some artifact accessory doesn't have any tag.

**Reply by microyahoo** [reply]
yes, the tags exists for busybox like below
![WX20240219-230313@2x](https://github.com/goharbor/harbor/assets/10202906/7b4455bf-6162-4e52-9c56-8eaded881b24)

**Comment by chlins** [comment, marked as answer]
Yes, the original webhook payload is not clear enough, and we resolved it in the v2.10, see https://github.com/goharbor/harbor/issues/19035 for reference.
